apps/server/minecraft_manager/audit/__init__.py

```
@@ -0,0 +1,7 @@
+"""Audit module — durable sanitized audit records and query port."""
+
+from .model import AuditRecord
+from .repository import SQLiteAuditRepository
+from .service import AuditService
+
+__all__ = ["AuditRecord", "SQLiteAuditRepository", "AuditService"]
```

Ruff (0.16.3)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)